UnitTest1.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Medicine;
+using NUnit.Framework;
+
+
+namespace TestProject11
+{
+    [TestFixture]
+    public class CommodityUnitTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var valerian = CreateTestItems();
+
+            Assert.AreEqual("Valerian", valerian.ProductName);
+            Assert.AreEqual("123456789", valerian.ItemNumber);
+            Assert.AreEqual("10$", valerian.Prise);
+            Assert.AreEqual("None", valerian.NumberOfGoods);
+            Assert.AreEqual(Medicine.withoutRecipe, valerian.Type);
+
+        }
+        [Test]
+        public void ToString_Items_ProductNameSpaceItemNumber()
+        {
+            var valerian = CreateTestItems();
+            Assert.AreEqual("Valerian 123456789", valerian.ToString());
+        }
+        private Stuff CreateTestItems()
+        {
+            return new Stuff("valerian", "123456789", "10", "None", Medicine.withoutRecipe);

```

Comment:
В методе подготовки данных для тестов аж 4 ошибки.

1. Перепутаны параметры конструктора
2. Неправильное название лекарства (в тестах проверяется другое).
3. Неправильно задается цена (в тестах другая).
4. Несуществующий тип данных

```suggestion
            return new Stuff("123456789", "Valerian", "10$", "None", Prescription.withoutRecipe);

```